packages/workflow/src/records/fhir.ts

```diff
@@ -1377,6 +1390,32 @@ export function toSavedBundle<T extends Resource>(
         }
       }
 
+      if (
+        isTask(entry.resource) &&
+        entry.resource.extension &&
+        entry.resource.extension.some((x) => x.url.includes('paymentDetails'))
```

Comment:
Can we match the full url `http://opencrvs.org/specs/extension/paymentDetails` here? Just so that when we use search to find references to it, this also pops up

Reply:
okay